nanofi/include/api/nanofi.h

```diff
@@ -135,16 +274,62 @@ flow_file_record* create_ff_object(const char *file, const size_t len, const uin
 
 flow_file_record* create_ff_object_na(const char *file, const size_t len, const uint64_t size);
 
-void free_flowfile(flow_file_record*);
+/**
+ * Get incoming flow file. To be used in processor logic callbacks.
+ * @param session current processor session
+ * @param context current processor context
+ * @return a flow file record or nullptr in case there is none in the session
+ **/
+flow_file_record* get_flowfile(processor_session* session, processor_context* context);
```

Comment:
So get has been replaced by get_flowfile ? Any reason we can't use this in python?

Reply:
I think we can, will give it a try